doc/user/backup_config.md

```diff
@@ -28,12 +28,24 @@ If configurations for both levels are specified then the cluster level configura
 
 ### Operator level configuration  
 
-See the [S3 backup deployment](../../example/deployment-s3-backup.yaml.template) template on how to configure the operator to enable S3 backups. The following flags need to be passed to operator:
+The operator supports AWS credential configuration in several ways.
+* Environment variables
+* Shared credentials file
+  * `backup-aws-secret` can be used to configure `~/.aws/credentials`
+  * `backup-aws-config` can be used to configure `~/.aws/config`
+* EC2 instance profiles
+
+See the [S3 backup deployment](../../example/deployment-s3-backup.yaml.template) template for an example of using shared credential files for the operator to enable S3 backups. 
+
+The following flags are supported for the operator:
 - `backup-aws-secret`: The name of the kube secret object that stores the AWS credential file. The file name must be 'credentials'.
-Profile must be "default".
```

Comment:
Because a user can use other values besides default?  As the AWS docs state if the user specifies `AWS_DEFAULT_PROFILE` environment variable they can use whatever profile they want.  I can add these lines back, but I don't believe it's a valid statement.